tests/integration_tests/test_ray.py

```diff
@@ -847,13 +847,14 @@ def test_ray_distributed_predict(tmpdir, ray_cluster_2cpu):
 
     with tempfile.TemporaryDirectory() as tmpdir:
         backend_config = {**RAY_BACKEND_CONFIG}
+        # Manually override num workers to 2 for distributed training and distributed predict
+        backend_config["trainer"]["num_workers"] = 2
```

Comment:
This actually overrides the value in `RAY)BACKEND_CONFIG`, since you're modifying a nested dict and only performed a shallow copy on line 849. So any tests that run after this will use 2 workers.

To fix, you can change line 849 to use `copy.deepcopy` instead.

Reply:
@tgaddair Ah, good catch, updated!